WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -866,14 +866,11 @@ private void updateFeaturedImageView() {
         // Get max width for photon thumbnail
         int width = DisplayUtils.getDisplayPixelWidth(getActivity());
         int height = DisplayUtils.getDisplayPixelHeight(getActivity());
-        int size = Math.max(width, height);
 
         String mediaUri = StringUtils.notNullStr(media.getThumbnailUrl());
```

Comment:
I think the real problem is that we're using `getThumbnailUrl()` for the image URL. Should this be using one of the larger image URLs instead?

Reply:
Well, that seemed ok but apparently `MediaModel` at that stage holds null stuff for all other sizes, look:

<img width="703" alt="screen shot 2019-01-24 at 12 29 51" src="https://user-images.githubusercontent.com/6597771/51688690-e0b48c00-1fd3-11e9-9e9c-64c51e9bb568.png">

And this is how it looks like, ofc:

<img width="300" alt="screen shot 2019-01-24 at 12 31 02" src="https://user-images.githubusercontent.com/6597771/51688724-f3c75c00-1fd3-11e9-9d65-755bf78ff0da.png">


I tested this with pics on the device, stock photos, etc.